### ISSUE TITLE
After update 1.0.13 plugin crashes

### ISSUE THREAD

**Issue by patric-eberle** [idok/scss-lint-plugin#17]
I've updated your plugin to Version 1.0.13 today. Unfortunately its not working anymore because of this error:
```
null
java.lang.NullPointerException
	at com.scss.annotator.Fixes.getFixForRule(Fixes.java:10)
	at com.scss.ScssLintExternalAnnotator.apply(ScssLintExternalAnnotator.java:117)
	at com.scss.ScssLintExternalAnnotator.apply(ScssLintExternalAnnotator.java:49)
	at com.intellij.codeInsight.daemon.impl.ExternalToolPass.b(ExternalToolPass.java:166)
	at com.intellij.codeInsight.daemon.impl.ExternalToolPass.access$600(ExternalToolPass.java:43)
	at com.intellij.codeInsight.daemon.impl.ExternalToolPass$1$1.run(ExternalToolPass.java:130)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1146)
	at com.intellij.codeInsight.daemon.impl.ExternalToolPass$1.run(ExternalToolPass.java:124)
	at com.intellij.util.ui.update.MergingUpdateQueue.execute(MergingUpdateQueue.java:320)
	at com.intellij.util.ui.update.MergingUpdateQueue.execute(MergingUpdateQueue.java:310)
	at com.intellij.util.ui.update.MergingUpdateQueue$2.run(MergingUpdateQueue.java:254)
	at com.intellij.util.ui.update.MergingUpdateQueue.flush(MergingUpdateQueue.java:269)
	at com.intellij.util.ui.update.MergingUpdateQueue.flush(MergingUpdateQueue.java:227)
	at com.intellij.util.ui.update.MergingUpdateQueue.run(MergingUpdateQueue.java:217)
	at com.intellij.util.concurrency.QueueProcessor.runSafely(QueueProcessor.java:238)
	at com.intellij.util.Alarm$Request$1.run(Alarm.java:351)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at com.intellij.util.concurrency.QueueProcessor$RunnableConsumer.consume(QueueProcessor.java:298)
	at com.intellij.util.concurrency.QueueProcessor$RunnableConsumer.consume(QueueProcessor.java:295)
	at com.intellij.util.concurrency.QueueProcessor$2$1.run(QueueProcessor.java:110)
	at com.intellij.util.concurrency.QueueProcessor.runSafely(QueueProcessor.java:238)
	at com.intellij.util.concurrency.QueueProcessor$2.consume(QueueProcessor.java:107)
	at com.intellij.util.concurrency.QueueProcessor$2.consume(QueueProcessor.java:104)
	at com.intellij.util.concurrency.QueueProcessor$3$1.run(QueueProcessor.java:215)
	at com.intellij.util.concurrency.QueueProcessor.runSafely(QueueProcessor.java:238)
	at com.intellij.util.concurrency.QueueProcessor$3.run(QueueProcessor.java:212)
	at com.intellij.openapi.application.impl.ApplicationImpl$8.run(ApplicationImpl.java:406)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:439)
	at java.util.concurrent.FutureTask$Sync.innerRun(FutureTask.java:303)
	at java.util.concurrent.FutureTask.run(FutureTask.java:138)
	at java.util.concurrent.ThreadPoolExecutor$Worker.runTask(ThreadPoolExecutor.java:895)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:918)
	at java.lang.Thread.run(Thread.java:695)
	at org.jetbrains.ide.PooledThreadExecutor$1$1.run(PooledThreadExecutor.java:56)
```

Updating scss-lint to version 0.39.0 had no effect.

**Comment by idok** [comment]
Does it happen with every scss file? what it the scss-lint config?
Which version of intellij do you use and which os?

**Comment by patric-eberle** [comment]
Sorry for the missing information.
Mac OS X 10.9
PhpStorm 8.0.3
The error seemed to occur on every file.
I'm not at the office today, so I don't have the scss-lint config. But there is a .scss-lint.yml file in the project root.

**Comment by patric-eberle** [comment]
This bug was fixed with the latest 1.0.14 release. thx!
